src/raftstore/store/worker/apply.rs

```diff
@@ -269,20 +346,21 @@ pub fn notify_stale_req(term: u64, cb: Callback) {
     cb.invoke_with_response(resp);
 }
 
-fn should_flush_to_engine(cmd: &RaftCmdRequest, wb_keys: usize) -> bool {
-    // When encounter ComputeHash cmd, we must flush the write batch to engine immediately.
+/// Check if a commit is needed to be issued before handle the command.
+fn should_commit_to_engine(cmd: &RaftCmdRequest, wb_keys: usize) -> bool {
```

Comment:
Can we use another name? There are too many `commit`.

Reply:
Do you have any suggestion?

Reply:
should_put/write_to_engine?